bin/btmon.py

```diff
@@ -1121,6 +1133,7 @@
 import traceback
 import urllib
 import urllib2
+from influxdb import InfluxDBClient
```

Comment:
could you modify the import so that influxdb is imported only in InfluxDBProcessor.process_calculated?

we don't want btmon to fail for people who do not use influxdb